Manager/src/test/java/io/raspberrywallet/manager/bitcoin/BitcoinTest.java

```diff
@@ -84,17 +85,51 @@ void getFreshReceiveAddress() throws WalletNotInitialized {
 
     @Test
     void getEstimatedBalance() throws WalletNotInitialized {
-        should_restore_from_file();
+        if (bitcoin.getWalletFile().length() > 0)
+            bitcoin.setupWalletFromFile(privateKeyHash, true);
+        else
+            bitcoin.setupWalletFromMnemonic(mnemonicCode, privateKeyHash, true);
         String balance = bitcoin.getEstimatedBalance();
         println(balance);
     }
 
     @Test
     void getAvailableBalance() throws WalletNotInitialized {
-        should_restore_from_file();
+        if (bitcoin.getWalletFile().length() > 0)
+            bitcoin.setupWalletFromFile(privateKeyHash, true);
+        else
+            bitcoin.setupWalletFromMnemonic(mnemonicCode, privateKeyHash, true);
         String availableBalance = bitcoin.getAvailableBalance();
         println(availableBalance);
     }
 
+    @Test
+    void sendFunds() throws WalletNotInitialized {
+        if (bitcoin.getWalletFile().length() > 0)
+            bitcoin.setupWalletFromFile(privateKeyHash, true);
+        else
+            bitcoin.setupWalletFromMnemonic(mnemonicCode, privateKeyHash, true);
+        String destinationAddress = "mk7exmQTiXjqzAMaxzBSHYezjkNvJGNGHX";
+        String amount = "0.0001";
+
+        bitcoin.sendCoins(amount, destinationAddress);
+    }
+
+    @Test
+    void printTransactions() throws WalletNotInitialized {
+        if (bitcoin.getWalletFile().length() > 0)
+            bitcoin.setupWalletFromFile(privateKeyHash, true);
+        else
+            bitcoin.setupWalletFromMnemonic(mnemonicCode, privateKeyHash, true);
+
+        final List<TransactionView> transactions = bitcoin.getAllTransactions();
+
+        int i = 1;
+        for (TransactionView tx : transactions) {
+            System.out.println(i + "  ________________________");
```

Comment:
shouldn't logger be used here instead of sout?

Reply:
Done